arxiv/main.tex

```diff
@@ -45,15 +45,18 @@
 \begin{abstract}
     This paper presents a new way of selecting an initial solution for the
     \(k\)-modes algorithm that allows for a notion of mathematical fairness and
```

Comment:
```suggestion
    \(k\)-modes algorithm that allows for a notion of game theoretic fairness and
```

how do you feel about that as a suggestion @gillardjw re your point about vagueness of `mathematical fairness`? I don't disagree that that's vague but perhaps `game theoretic` is slightly more precise?